backend/dataall/cdkproxy/cdk_cli_wrapper.py

```diff
@@ -108,6 +106,24 @@ def deploy_cdk_stack(engine: Engine, stackid: str, app_path: str = None, path: s
                     }
                 )
 
+            if stack.stack == 'cdkpipeline':
```

Comment:
If we ended up having many extensions we would have to do something like if `stack.stack == stack_name` for all of them. I'd rather replace it with:
```
extension = _CDK_CLI_WRAPPER_EXTENSIONS.get(stack.stack)
if extension:
    do stuff
else:
    log.info(f"There is no extension for {stack.stack}. Proceeding further...")

```

Reply:
I agree with this. Even with the extension, the code is still very specific to `cdkpipeline` extension

Reply:
Goog point, I added @nikpodsh 's suggestion